README.md

```diff
@@ -313,19 +313,13 @@ Finally: your plugin's `package.json` must expose each UI extension as a sub-mod
     // Main module entry point – the default Astro Integration export
     ".": "./dist/index.js",
     // Module export for the HelloWorldMessage UI extension
-    "./HelloWorldMessage": "./dist/extensions/HelloWorldMessage.js"
+    "./HelloWorldMessage": "./dist/HelloWorldMessage.js"
```

Comment:
Or this. Above, we created the React component at the path `src/HelloWorldMessage.tsx`, which will compile it to `./dist/HelloWorldMessage.js`. So either we tweak the path in the exports (as I've done), or we need to create the React component in `src/extensions/HelloWorldMessage.tsx`.